webv2/updateTableSchema/removeColumn.go

```diff
@@ -0,0 +1,331 @@
+package updateTableSchema
+
+import (
+	"fmt"
+
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+	utilities "github.com/cloudspannerecosystem/harbourbridge/webv2/utilities"
+)
+
+// removeColumn remove given column from schema.
+func removeColumn(table string, colName string, Conv *internal.Conv) {
+
+	sp := Conv.SpSchema[table]
+
+	removeColumnFromCurrentTableSchema(Conv, sp, table, colName)
+
+	// update foreignKey relationship Table column names
+	for i, _ := range sp.Fks {
+
+		removeColumnFromForeignkeyTableSchema(Conv, sp, i, colName)
+
+	}
+
+	for _, sp := range Conv.SpSchema {
+
+		for j := 0; j < len(sp.Fks); j++ {
+			if sp.Fks[j].ReferTable == table {
+				removeColumnFromForeignkeyReferTableSchema(Conv, sp, sp.Name, colName)
+			}
+
+		}
+
+	}
+
+	isParent, parentSchemaTable := IsParent(table)
+
+	if isParent {
+
+		removeColumnFromParentTableSchema(Conv, parentSchemaTable, colName)
+
+	}
+
+	childSchemaTable := Conv.SpSchema[table].Parent
+
+	if childSchemaTable != "" {
+
+		removeColumnFromChildTableSchema(Conv, parentSchemaTable, colName)
+	}
+}
+
+// removeColumnFromCurrentTableSchema remove given column from current table schema.
+func removeColumnFromCurrentTableSchema(Conv *internal.Conv, sp ddl.CreateTable, table string, colName string) {
```

Comment:
Use this method in a generic way instead of creating 5 separate methods

Reply:
Created a common function - removeColumFromTableSchema